contentgrid-spring-boot-autoconfigure/src/main/java/com/contentgrid/spring/boot/autoconfigure/ContentGrid.java

```diff
@@ -0,0 +1,17 @@
+package com.contentgrid.spring.boot.autoconfigure;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+
+@Target({ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER, ElementType.TYPE, ElementType.ANNOTATION_TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+@Qualifier
+public @interface ContentGrid {
```

Comment:
wait, what does this do ? :upside_down_face:

Reply:
It's a bean qualifier annotation.

For the `MessageConverter`s, I am creating beans that implement a Spring framework interface. I don't want other places that also register beans to interfere with these.

Reply:
Discussed: renaming to `@ContentGridMessaging` and moving to the `.messaging` package would be more appropriate